### PR TITLE
Add WhiteSource CVEs scan to Github workflow

### PR DESCRIPTION
**Pull request by zelinh**
### Description
Refer to #1157, WhiteSource integration may not work correctly at this time. They have an issue on their side. We are switching to our Plan B using Github Actions for automating WhiteSource scan using old [script](https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/.github/workflows/check-vulnerability-whitesource.yml) from ODFE. This workflow will update the scan results in our testing product on WhiteSource dashboard first so that it won't overwrite the previous manual scan results that team tracked. 
 
### Issues Resolved
Backup plan for #1157 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — CHANGES_REQUESTED** [review]
Is there an example of this workflow that I could see?  I'm not sure that this has executed on your fork

File: .github/workflows/check-vulnerability-whitesource.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_WSS_EC2_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_WSS_EC2_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Setting up runner
```
Comment (gaiksaya) [inline .github/workflows/check-vulnerability-whitesource.yml:21]: We have stopped using provision runners for GHA. So this wont work. Not a recommended approach

**Comment by zelinh** [comment]
> Is there an example of this workflow that I could see? I'm not sure that this has executed on your fork

This is using secret that stored in build repo. I can't run it on my fork. We had same workflow before here. https://github.com/opensearch-project/opensearch-build/blob/41064bc5d3cebed31d06eab2c88d12462e141569/.github/workflows/check-vulnerability-whitesource.yml

File: .github/workflows/check-vulnerability-whitesource.yml (hunk 1 above)
Comment (zelinh) [reply .github/workflows/check-vulnerability-whitesource.yml:21]: Do we have substitute for this?

Comment (gaiksaya) [reply .github/workflows/check-vulnerability-whitesource.yml:21]: As said above. Jenkins

**Comment by peternied** [comment]
> > Is there an example of this workflow that I could see? I'm not sure that this has executed on your fork
> 
> This is using secret that stored in build repo. I can't run it on my fork. We had same workflow before here. https://github.com/opensearch-project/opensearch-build/blob/41064bc5d3cebed31d06eab2c88d12462e141569/.github/workflows/check-vulnerability-whitesource.yml

That change was executed months ago, lets re-run everything with the latest and greatest to make sure everything is still working well.

**Comment by dblock** [comment]
Regardless of Jenkins vs. GhA, shouldn't this live in the OpenSearch repo?

**Comment by zelinh** [comment]
We are closing this one since this approach won't work now.

**Pull request closed without merging**